SecondClasses/quicksort/src/quicksort.erl

```diff
@@ -1,6 +1,16 @@
 -module(quicksort).
 
--export([sort/1]).
+-export([sort/1, ordered/1]).
 
-sort(L) ->
-    1.
+sort([]) -> [];
+sort([H | T]) ->
+    {S, B} = divide(H, T),
+    sort(S) ++ [H] ++ sort(B).
+
+divide(V, L) -> lists:foldl(
+    fun
+        (X ,{L1, L2}) when X < V -> {L1 ++ [X], L2};
```

Comment:
Since the order of subsets `L1` and `L2` does not matter I would rather recommend:
```erlang
        (X ,{L1, L2}) when X < V -> {[ X | L ], L2};
        (X ,{L1, L2}) when X >= V -> {L1, [X | L2]}
```
For explanation see [This document](http://erlang.org/doc/efficiency_guide/listHandling.html#creating-a-list)

Even if you need to keep the order it is better to generate the list in reversed order and when returning the list reversing it once.